server.h

```diff
@@ -129,13 +127,22 @@ class CTFOServer final {
     event_log_stream_.open(event_log_file_, std::ofstream::out | std::ofstream::app);
 
     bricks::cerealize::CerealFileParser<Card, bricks::cerealize::CerealFormat::JSON> cf(cards_file);
+    const UID admin_uid = static_cast<UID>(1000000000000000001ull);
```

Comment:
I suppose, admin user could be used in many places, so maybe define it more globally?